tests/Dfc.CourseDirectory.Testing/TestData/TestData.CreateApprenticeship.cs

```diff
@@ -16,7 +17,8 @@ public partial class TestData
             string contactTelephone = "01234 567890",
             string contactEmail = "admin@provider.com",
             string contactWebsite = "http://provider.com",
-            DateTime? createdUtc = null)
+            DateTime? createdUtc = null,
+            Func<IEnumerable<CreateApprenticeshipLocation>> Locations =null)
```

Comment:
`IEnumerable<CreateApprenticeshipLocation>` instead of `Func<>` (unless there's a reason it needs to be deferred) and nit: `locations` instead of `Locations`

Reply:
changed, no reason for it to be a func.